src/ServiceControl.Config/Features.cs

```diff
@@ -50,6 +50,7 @@ public FeatureToggleDefaults(FeatureToggles featureToggles)
 
         public void Start()
         {
+            featureToggles.Enable(Feature.MonitoringInstances);
```

Comment:
Why not simply remove the `ShowMonitoringInstances` checks that we have, assuming that we will always show monitoring instances in perpetuity?

Reply:
If we leave it in, it's easy to reverse it if something goes wrong. Once everything is stable we can remove the toggle.

Reply:
We can achieve the same with a revert commit.

We could also publish a default app.config that has the config flag enabled. In fact, I might prefer this option instead of hard-coding it on or off if we're not sure how long until it's stable